pytest_logger/plugin.py

```diff
@@ -195,6 +211,15 @@ def set_log_option_default(self, value):
         """ Sets default value of `log` option."""
         self._log_option_default = value
 
+    def divide_logs_by_result(self, div_by_res_logdir='div_by_res', div_by_res_subdirs=['failed']):
+        """Makes a directory inside main logdir where logs are further divided by test result
```

Comment:
`test result` -> `outcome`, test result is too broad.

Reply:
done